packages/arb-validator/cmd/rollupServer/rollupServer.go

```diff
@@ -159,6 +161,7 @@ func validateRollupChain() error {
 
 	// Rollup creation
 	auth := bind.NewKeyedTransactor(key)
+	auth.GasPrice = big.NewInt(*gasPrice)
```

Comment:
The GasPrice field is denominated in Wei, but this uses a value in Gwei

Reply:
Fixed. Now take commandline option as a float64 in Gwei, which gets converted to Wei.